Application Actions/Firefox/1.0/default.json

```diff
@@ -1,11 +1,11 @@
 {
-  "ButtonPress": "New Tab",
-  "SwipeLeft":   "Select Previous Tab",
-  "SwipeRight":  "Select Next Tab",
-  "SwipeUp":     "Select Previous Tab",
-  "SwipeDown":   "Select Next Tab",
-  "RotateLeft":  "Select Previous Tab",
-  "RotateRight": "Select Next Tab",
-  "FlyLeft":     "Select Previous Tab",
-  "FlyRight":    "Select Next Tab",
+  "ButtonPress"	:	"New Tab",
+  "SwipeLeft"	:	"Select Previous Tab",
+  "SwipeRight"	:	"Select Next Tab",
+  "SwipeUp"		:	"Reopen Closed Tab",
+  "SwipeDown"	:	"Close Tab",
+  "RotateLeft"	:	"Scroll Down",
+  "RotateRight"	:	"Scroll Up",
+  "FlyLeft"		:	"Select Previous Tab",
+  "FlyRight"	:	"Select Next Tab",
```

Comment:
Can you please preserve the formatting? The actions shall be left-aligned. Please don't use tabs but spaces.